### ISSUE TITLE
Auto-Installation is broken on AGP 7.2.0

### ISSUE THREAD

**Issue by romtsn** [getsentry/sentry-android-gradle-plugin#342]
### Gradle Version

7.4

### AGP Version

7.2.0

### Version

3.1.2

### Sentry SDK Version

6.1.0

### Steps to Reproduce

IDE is failing to sync when using our Plugin with Auto-Install enabled and AGP 7.2.0. This is an issue on the AGP side, which was fixed on 7.2.1 and caused by circular dependencies on POM files (that's what we do in order to achieve auto-installation).
https://issuetracker.google.com/issues/232075280

### Expected Result

Should sync successfully

### Actual Result

IDE Sync fails

**Comment by bruno-garcia** [comment]
@romtsn sounds like all we need here is an entry in our [troubleshooting page](https://docs.sentry.io/platforms/android/troubleshooting/)?

**Comment by romtsn** [comment]
@bruno-garcia Yeah or we disable the auto-installation for 7.2.0 and warn the user?

The proper fix for this would be, we switch to [compileOnly](https://github.com/getsentry/sentry-java/issues/2125) dependencies in our integrations - this way we wouldn't have a circular dependency on the POM file, and the issue wouldn't exist. But I guess as an intermediate step we can just have a troubleshooting page until we properly fix it.

**Comment by bruno-garcia** [comment]
> @bruno-garcia Yeah or we disable the auto-installation for 7.2.0 and warn the user?

Definitely, if that works, good idea.

**Comment by romtsn** [comment]
Fixed by #357
